src/lib/llm/LangChainProvider.ts

```
@@ -58,177 +46,217 @@ export class LangChainProvider {
+      apiKey: provider.apiKey,
+      convertSystemMessageToHumanContent: true
+    })
   }
   
-  private _getCacheKey(config: LLMConfig): string {
-    return `${config.provider}-${config.model}-${config.temperature}-${config.maxTokens || 'default'}`
+  // Ollama provider (local, no API key required)
+  private _createOllamaLLM(
+    provider: BrowserOSProvider,
+    temperature: number,
+    maxTokens?: number
```

ESLint

[error] 'maxTokens' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)